### ISSUE TITLE
New Bash devfile is failing to start - failed to download plugin

### ISSUE THREAD

**Issue by ibuziuk** [eclipse-che/che#18940]
Che version 7.25.0 
```
Loading...
Successfully assigned test-username-che/workspacenrbmckp4d9d8rcdw.che-plugin-broker to ip-172-31-64-40.us-east-2.compute.internal
pulling image "quay.io/eclipse/che-plugin-metadata-broker:v3.4.0"
Successfully pulled image "quay.io/eclipse/che-plugin-metadata-broker:v3.4.0"
Created container
Started container
Starting plugin metadata broker
All plugin metadata has been successfully processed
List of plugins and editors to install
- mads-hartmann/bash-ide-vscode/latest - A language server for Bash
- rogalmic/bash-debug/latest - A debugger extension for bash scripts (using bashdb).
- timonwong/shellcheck/latest - An extension to use shellcheck in vscode
- eclipse/che-workspace-telemetry-woopra-plugin/0.0.1 - Telemetry plugin to send information to Woopra
- eclipse/che-machine-exec-plugin/7.25.0 - Che Plug-in with che-machine-exec service to provide creation terminal or tasks for Eclipse Che workspace containers.
- eclipse/che-theia/latest - Eclipse Theia
pulling image "quay.io/eclipse/che-theia-endpoint-runtime-binary:7.25.0"
Successfully pulled image "quay.io/eclipse/che-theia-endpoint-runtime-binary:7.25.0"
Created container
Started container
pulling image "quay.io/eclipse/che-plugin-artifacts-broker:v3.4.0"
Successfully pulled image "quay.io/eclipse/che-plugin-artifacts-broker:v3.4.0"
Created container
Started container
Starting plugin artifacts broker
Cleaning /plugins dir
Processing plugin mads-hartmann/bash-ide-vscode/latest
  Installing plugin extension 1/1
    Downloading plugin from https://open-vsx.org/api/mads-hartmann/bash-ide-vscode/1.10.2/file/mads-hartmann.bash-ide-vscode-1.10.2.vsix
Processing plugin rogalmic/bash-debug/latest
  Installing plugin extension 1/1
    Downloading plugin from https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix
failed to download plugin from https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix: Downloading https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix failed. Status code 404
pulling image "quay.io/eclipse/che-plugin-artifacts-broker:v3.4.0"
Successfully pulled image "quay.io/eclipse/che-plugin-artifacts-broker:v3.4.0"
Created container
Started container
Starting plugin artifacts broker
Cleaning /plugins dir
Processing plugin mads-hartmann/bash-ide-vscode/latest
  Installing plugin extension 1/1
    Downloading plugin from https://open-vsx.org/api/mads-hartmann/bash-ide-vscode/1.10.2/file/mads-hartmann.bash-ide-vscode-1.10.2.vsix
Processing plugin rogalmic/bash-debug/latest
  Installing plugin extension 1/1
    Downloading plugin from https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix
failed to download plugin from https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix: Downloading https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix failed. Status code 404
Back-off restarting failed container
pulling image "quay.io/eclipse/che-plugin-artifacts-broker:v3.4.0"
Successfully pulled image "quay.io/eclipse/che-plugin-artifacts-broker:v3.4.0"
Created container
Started container
Starting plugin artifacts broker
Cleaning /plugins dir
Processing plugin mads-hartmann/bash-ide-vscode/latest
  Installing plugin extension 1/1
    Downloading plugin from https://open-vsx.org/api/mads-hartmann/bash-ide-vscode/1.10.2/file/mads-hartmann.bash-ide-vscode-1.10.2.vsix
Processing plugin rogalmic/bash-debug/latest
  Installing plugin extension 1/1
    Downloading plugin from https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix
failed to download plugin from https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix: Downloading https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix failed. Status code 404
Back-off restarting failed container
Back-off restarting failed container
pulling image "quay.io/eclipse/che-plugin-artifacts-broker:v3.4.0"
Successfully pulled image "quay.io/eclipse/che-plugin-artifacts-broker:v3.4.0"
Created container
Started container
Starting plugin artifacts broker
Cleaning /plugins dir
Processing plugin mads-hartmann/bash-ide-vscode/latest
  Installing plugin extension 1/1
    Downloading plugin from https://open-vsx.org/api/mads-hartmann/bash-ide-vscode/1.10.2/file/mads-hartmann.bash-ide-vscode-1.10.2.vsix
Processing plugin rogalmic/bash-debug/latest
  Installing plugin extension 1/1
    Downloading plugin from https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix
failed to download plugin from https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix: Downloading https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix failed. Status code 404
Back-off restarting failed container
Back-off restarting failed container
Starting plugin artifacts broker
Cleaning /plugins dir
Processing plugin mads-hartmann/bash-ide-vscode/latest
  Installing plugin extension 1/1
    Downloading plugin from https://open-vsx.org/api/mads-hartmann/bash-ide-vscode/1.10.2/file/mads-hartmann.bash-ide-vscode-1.10.2.vsix
Processing plugin rogalmic/bash-debug/latest
  Installing plugin extension 1/1
    Downloading plugin from https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix
```
failed to download plugin from https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix: Downloading https://open-vsx.org/api/rogalmic/bash-debug/0.3.9/file/rogalmic.bash-debug-0.3.9.vsix failed. Status code 404

**Comment by ibuziuk** [comment]
ok, looks like related to https://github.com/eclipse/che/issues/18933

**Comment by ibuziuk** [comment]
@ericwill could you please make sure that we have a fix for that cherry-picked in 7.25.x ? would be great to have it in 7.25.2 that is planned for tomorrow

**Comment by ericwill** [comment]
> @ericwill could you please make sure that we have a fix for that cherry-picked in 7.25.x ? would be great to have it in 7.25.2 that is planned for tomorrow

Cherry pick PR: https://github.com/eclipse/che-plugin-registry/pull/829

**Comment by ericwill** [comment]
> @ericwill could you please make sure that we have a fix for that cherry-picked in 7.25.x ? would be great to have it in 7.25.2 that is planned for tomorrow

Done!

**Comment by ScrewTSW** [comment]
Verified working in 7.25.2
